app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormatExpr.scala

```diff
@@ -502,3 +502,10 @@ object TextExpression {
 
   implicit val equal: Eq[TextExpression] = Eq.fromUniversalEquals
 }
+
+object MiniSummaryListValue {
+  sealed trait MiniSummaryListValue
+  case class MiniSummaryListExpr(expr: Expr) extends MiniSummaryListValue
```

Comment:
Since this is qualified by `MiniSummaryListValue` can we call it just `Expr`? And also can we remove all
```
import MiniSummaryListValue._
import MiniSummaryList._
```